server/mappers/headerMappers.ts

```diff
@@ -15,6 +15,8 @@ export function mapProfileBannerTopLinks(prisonerData: Prisoner, user: User) {
   const { userRoles, caseLoads } = user
   const profileBannerTopLinks = []
 
+  const belongsToCaseLoad = prisonerBelongsToUsersCaseLoad(prisonerData.prisonId, user.caseLoads)
+
   if (prisonerBelongsToUsersCaseLoad(prisonerData.prisonId, caseLoads)) {
```

Comment:
Should swap use the newly defined `belongsToCaseLoad` in this `if` condition rather than calling the method again.